stingray/crossspectrum.py

```diff
@@ -1,19 +1,23 @@
-from __future__ import division
+from __future__ import division, absolute_import, print_function
 
 import numpy as np
 import scipy
 import scipy.stats
 import scipy.fftpack
 import scipy.optimize
 
-import stingray.lightcurve as lightcurve
-import stingray.utils as utils
-from stingray.exceptions import StingrayError
-from stingray.gti import cross_two_gtis, bin_intervals_from_gtis, check_gtis
+from .lightcurve import Lightcurve
+from .utils import rebin_data, simon
+from .exceptions import StingrayError
+from .gti import cross_two_gtis, bin_intervals_from_gtis, check_gtis
 
```

Comment:
Oh, I didn't know about PEP-8. The modification was actually an experiment, because I had the impression that tests were run with the installed version and not the current code if I didn't do that (it turned out not to be true). No strong opinions about it. Just curious, why are relative imports discouraged?